src/test-apps/happy/lib/WeaveDeviceManager.py

```diff
@@ -32,7 +32,20 @@
 import base64
 import json
 import set_test_path
-import WeaveDeviceMgr
+
+try:
+    import WeaveDeviceMgr
+except Exception:
+    weaveDeviceMgrPath = os.path.normpath(os.path.join(os.path.dirname(os.path.abspath(__file__)), "..", "..", "..", "..", "src", "device-manager", "python"))
```

Comment:
This seems really dodgy.  Is there a more readable way of finding an appropriate the paths?

Reply:
it seems it is a little difficult on it.
Looking at the below path, in build folder, weave-device-mgr is linking to  /usr/local/openweave-core/src/device-manager/python/weave-device-mgr.py, and there is no WeaveDeviceManager folder, then it seems we need use WeaveDeviceManager from openweave-core/src/device-manager/python/WeaveDeviceManager.py.
build/x86_64-unknown-linux-gnu/src/device-manager/python$ ls -al weave-device-mgr
weave-device-mgr -> /usr/local/openweave-core/src/device-manager/python/weave-device-mgr.py

Further, the so file has been stored in .libs, it seems we need to add this so path via adding dmLibPath in system path.
openweave-core/build/x86_64-unknown-linux-gnu/src/device-manager/python/.libs$ ls
_WeaveDeviceMgr.a   _WeaveDeviceMgr.lai                                             _WeaveDeviceMgr_la-WeaveDeviceManager-BlePlatformDelegate.o  _WeaveDeviceMgr.so
_WeaveDeviceMgr.la  _WeaveDeviceMgr_la-WeaveDeviceManager-BleApplicationDelegate.o  _WeaveDeviceMgr_la-WeaveDeviceManager-ScriptBinding.o

Reply:
Done, move these paths in weave.py and make this a little better on readability in weave.py, and generate paths based upon self.weave_path

t